docs/source/extensions_tutorial/5_custom_api.rst

```diff
@@ -48,13 +92,13 @@ Alternatively, you can use :py:func:`~pynwb.register_class` as a decorator.
 :py:class:`~pynwb.core.NWBContainer`.
 
 
-``__nwbfields__``
------------------
+Nwbfields
+---------
 
-When subclassing :py:class:`~pynwb.core.NWBData` or :py:class:`~pynwb.core.NWBContainer`, you might want to
-define some properties on your class. This can be done using the ``__nwbfields__`` class property. This
-class property should be a tuple of strings that name the properties. Adding a property using this functionality
-will create a property than can be set *only once*.
+When creating a new neurodata type, you need to define the new properties on your class, which is done by defining
+them in the ``__nwbfields__`` class property. This class property should be a tuple of strings that name the new
```

Comment:
@rly should this be changed to ``__fields__`` instead of ``__nwbfields__``?

Reply:
Not yet